source/security/private-endpoints.txt

```diff
@@ -139,21 +140,33 @@ connection using either the App Services UI or the Admin API.
             value starts with ``vpce-``.
 
             Once you have the **VPC Endpoint ID**, enter it in the input box.
-            Then, click :guilabel:`Next`.
 
-         .. step:: Modify Your VPC Private DNS
+            Optionally, enter a comment that describes the purpose of the
+            endpoint. Then click :guilabel:`Next`.
 
-            Use the AWS CLI to modify your VPC private DNS settings. This
-            ensures that your App can resolve the private endpoint's DNS name
-            to the correct IP address.
-
-            .. code-block:: sh
-
-               aws ec2 modify-vpc-endpoint \
-                 --region <your-aws-deployment-region> \
-                 --vpc-endpoint-id <your-vpc-endpoint-id> \
-                 --private-dns-enabled
+         .. step:: Modify Your VPC Private DNS
 
+            a. Use the AWS CLI to modify your VPC private DNS settings. This
+               ensures that your App can resolve the private endpoint's DNS name
+               to the correct IP address.
+   
+               .. code-block:: sh
+   
+                  aws ec2 modify-vpc-endpoint \
+                    --region <your-aws-deployment-region> \
+                    --vpc-endpoint-id <your-vpc-endpoint-id> \
+                    --private-dns-enabled
+            
+            #. If this is the first Private Endpoint you are creating, you will 
+               see a toggle you can use to either restrict or open all access from 
+               requests outside your PrivateLink connection. By default, App 
+               Services blocks all requests coming from outside your PrivateLink 
+               connection. This setting applies to this endpoint and **all 
+               additional endpoints you create**. Refer to :ref:`change access 
+               restrictions <change_access_restrictions>` for more information.
```

Comment:
Can we add an equivalent step to the Admin API procedure step 2? i.e. a new field in the request body (if possible)

Reply:
There is no new field in the request body, alas. It can only be set via the new endpoint.